assets/js/components/user-input/UserInputQuestionnaire.js

```diff
@@ -107,6 +148,9 @@ export default function UserInputQuestionnaire() {
 					questionNumber={ 1 }
 					title={ __( 'Which best describes your team/role in relation to this site?', 'google-site-kit' ) }
 					description={ __( 'This will help Site Kit show tips that help you specifically in your role.', 'google-site-kit' ) }
+					single={ single }
+					submitChanges={ submitChanges }
+					goToPreview={ goToPreview }
```

Comment:
@benbowler instead of passing it to every wrapper, let's leverage `next` and `nextLabel` props. It can be done like this:

```javascript
let nextCallback = next;
let nextLabel = undefined;

if ( single === 'user-input' ) {
    nextCallback = goToPreview;
    nextLabel = __( 'Update', 'google-site-kit' );
} else if ( single === 'settings' ) {
    nextCallback = submitChanges;
    nextLabel = __( 'Submit', 'google-site-kit' );
}

...

return (
    ...
    <UserInputQuestionWrapper
        next={ nextCallback }
        nextLabel={ nextLabel }
    />
    ...
);
```

> For the last question you can use `nextLabel={ nextLabel || __( 'Preview', 'google-site-kit' ) }` to make sure it works correctly.

In this case, we won't need additional logic in the wrapper component.